server.py

```diff
@@ -1537,7 +1537,7 @@ def get(self, asset_id):
             result = {
                 'type': 'file',
                 'filename': filename,
-                'content': b64encode(content),
+                'content': b64encode(content).decode(),
```

Comment:
looks aggressive to me. We definitely need a unit test for this.

Reply:
Got it. I'll definitely create an issue and a PR for this one. Thanks!